src/scene_graph/camera_node.hpp

```diff
@@ -14,15 +14,24 @@ namespace wr
 			m_active(true),
 			m_frustum_near(0.1f),
 			m_frustum_far(1000.0f),
-			m_fov(fov_deg / 180 * 3.1415926535f),
-			m_aspect_ratio(aspect_ratio)
+			m_fov(2.0f * std::atan2(35 / aspect_ratio, 2.0f * 30.f)),
```

Comment:
Replace this constructor with 2 constructors. both with a different named type for FoV. So its immediately obvious what fov calculation we are using.